gamestonk_terminal/cryptocurrency/discovery/pycoingecko_model.py

```diff
@@ -0,0 +1,451 @@
+import pandas as pd
+from pycoingecko import CoinGeckoAPI
+from gamestonk_terminal.cryptocurrency.cryptocurrency_helpers import (
+    wrap_text_in_df,
+    percent_to_float,
+    create_df_index,
+)
+from gamestonk_terminal.cryptocurrency.pycoingecko_helpers import (
+    changes_parser,
+    replace_qm,
+    clean_row,
+    collateral_auditors_parse,
+    swap_columns,
+    scrape_gecko_data,
+    get_btc_price,
+    GECKO_BASE_URL,
+)
+
+PERIODS = {
+    "1h": "?time=h1",
+    "24h": "?time=h24",
+    "7d": "?time=d7",
+    "14d": "?time=d14",
+    "30d": "?time=d30",
+    "60d": "?time=d60",
+    "1y": "?time=y1",
+}
+
+CATEGORIES = {
+    "trending": 0,
+    "most_voted": 1,
+    "positive_sentiment": 2,
+    "recently_added": 3,
+    "most_visited": 4,
+}
+
+COLUMNS = {
```

Comment:
This COLUMNS is almost exactly the same, I would do something different. Just because the only different I can see is the last_added: added